### DISCUSSION TITLE
SDK: Warn sdk version != directus version

### DISCUSSION THREAD

**Discussion by MiniDigger** [directus/directus#12214] [Feature Requests]
It would be cool if the SDK could somehow detect the directus version it is connecting too and print a warning if it doesn't match, as mismatched versions can cause issues, especially when people look at outdated/updated docs that don't match their SDK version :D

**Comment by rijkvanzanten** [comment]
That'd be cool! It's a curious design challenge though, as there's a couple issues we'd have to overcome. The first is that the Directus installation will only share its current version when you're an authenticated user, so we can't always get that information ahead of time. The second thing is that it might not be wanted that the SDK starts making requests without the user's direct command/consent 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
